### PR TITLE
Feature : Backend for BasicCarInfo Completed

### PR DESCRIPTION
**Pull request by harismalik-1**
# Description

BasicCarInfo backend completed, now it display the basic car information stored in the database and if it doesn't exist it gives a toast message that the car does not exist in the database.

Fixes # (issue)

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: app/src/main/res/layout/activity_basic_car_info.xml [hunk 1]
```diff
@@ -30,59 +30,62 @@
 
         android:orientation="horizontal">
 
-        <EditText
+        <TextView
```
Comment (l0minous) [inline app/src/main/res/layout/activity_basic_car_info.xml:33]: Correctly changes to Text View as discussed

File: app/src/main/java/com/course_project_autotrace/RegisterNewVehicle.java [hunk 2]
```diff
@@ -55,7 +55,7 @@ public void onDataChange(@NonNull DataSnapshot dataSnapshot) {
                     }
 
                     if (car != null) {
-                        referenceToUsers.child(userID).child(Objects.requireNonNull(car.getKey())).setValue(car.getValue()).addOnSuccessListener(aVoid -> {
+                        referenceToUsers.child(userID).child("Cars").child(Objects.requireNonNull(car.getKey())).setValue(car.getValue()).addOnSuccessListener(aVoid -> {
```
Comment (l0minous) [inline app/src/main/java/com/course_project_autotrace/RegisterNewVehicle.java:58]: Edits reference in the registration screen to comply with the updated database sorting system

File: app/src/main/java/com/course_project_autotrace/LoginScreen.java [hunk 3]
```diff
@@ -9,17 +9,29 @@
...
 //import org.w3c.dom.Text;
 
 public class LoginScreen extends AppCompatActivity {
     private FirebaseAuth mFirebaseAuth; // for firebase autheniation
     private EditText mEtEmail; //for edit text.
     private EditText mEtPwd;
+    public EditText licenseNum;
+    private DatabaseReference referenceToCars;
+    private DataSnapshot car;
+    public String carName, model,info, insurance;
```
Comment (l0minous) [inline app/src/main/java/com/course_project_autotrace/LoginScreen.java:33]: correctly splits items into public and private

**Review by l0minous — APPROVED** [review]
Looks great overall; I tested the files, and they work.

**Comment by milan-panta** [comment]
Looks good.

**Review by milan-panta — APPROVED** [review]

**Review by l0minous — APPROVED** [review]

**Pull request merged**